gatsby/gatsby-plugin-sitemap/gatsby-plugin-sitemap.ts

```diff
@@ -162,18 +162,15 @@ export const serialize = (sitemapItem) => {
   };
 };
 
-export const filterPages = ({ path }, excludes) => {
+/**
+ * Gets called for every excluded path on every page.
+ */
+export const filterPages = ({ path }, excludedPath) => {
   const [EXCLUDE, KEEP] = [true, false];
 
-  if (excludes === path) {
+  if (excludedPath === path) {
     return EXCLUDE;
   }
 
-  if (!path.endsWith('/')) {
-    console.warn(
-      `Path '${path}' does not end with a slash! For SEO reasons all paths should end with a slash`,
```

Comment:
We don't need this warning anymore. It's also not the best place to check the URLs, as the filter is called multiple times.

Reply:
🙌